controllers/show.go

```diff
@@ -120,21 +120,36 @@ func (sc *ShowController) GetTimeslot(w http.ResponseWriter, r *http.Request) {
 	id, _ := strconv.Atoi(vars["id"])
 
 	timeslot, tracklist, creditsToUsers, err := sm.GetTimeslot(id)
-	mixcloudavailable := false
-
-	if strings.HasPrefix(timeslot.MixcloudStatus, "/URY1350/") {
-		mixcloudavailable = true
+	odState := 0
+
+	endTime := timeslot.StartTime.Add(timeslot.Duration)
+	timeNow := time.Now()
+	if timeNow.Before(timeslot.StartTime) {
+		odState = 1 // Show hasn't started yet
```

Comment:
It'd be nice to use an enum here instead of magic numbers, but i suspect that'd make it difficult to get the values out of the template

Reply:
Yeah, I haven't seen much of a way of doing that in the templates...